README.md

```diff
@@ -56,6 +58,73 @@ you can still submit your content as a Jupyter Notebook file.
 If you have your own tutorial in the form of a Jupyter notebook (a `.ipynb`
 file) and you'd like to add it to the repository:
 
+#### Create or pair your [`myst`](https://jupytext.readthedocs.io/en/latest/formats.html#myst-markdown) notebook
+
+The NumPy tutorials are reviewed and executed as a
+[MyST-NB](https://myst-nb.readthedocs.io/) notebooks. The benefit is that
+content is easier to review in this markdown format. You can convert or
+pair your `.ipynb` to `myst` using
+[Jupytext](https://jupytext.readthedocs.io/en/latest/index.html).
+Install `jupytext` using:
+
+```
+pip install jupytext
+```
+or
+```
+conda install jupytext -c conda-forge
+```
+
+Once installed, you can start your `jupyter lab` or `jupyter notebook`
+in the browser. When launching `jupyter lab` it will ask you to rebuild
+to include the Jupytext extension. 
```

Comment:
```suggestion
Once installed, you can start your `jupyter lab` or `jupyter notebook`
session in the browser. When launching `jupyter lab` it will ask you to rebuild
to include the Jupytext extension. 
```